financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsConsumerSessionRepository.kt

```diff
@@ -27,17 +27,23 @@ internal interface FinancialConnectionsConsumerSessionRepository {
 
     suspend fun getCachedConsumerSession(): CachedConsumerSession?
 
+    suspend fun postConsumerSession(
+        email: String,
+        clientSecret: String
+    ): ConsumerSessionLookup
+
+    suspend fun mobileLookupConsumerSession(
+        email: String,
+        verificationToken: String,
+        appId: String
+    ): ConsumerSessionLookup
+
     suspend fun signUp(
         email: String,
         phoneNumber: String,
         country: String,
     ): ConsumerSessionSignup
 
-    suspend fun lookupConsumerSession(
-        email: String,
-        clientSecret: String
-    ): ConsumerSessionLookup
-
```

Comment:
this was wrongly called lookup, we were actually calling `POST consumer_session`. Renamed accordingly.